examples/custom-logger/custom-logger.cpp

```diff
@@ -232,7 +232,7 @@ int main(int argc, char *argv[])
                gko::HipExecutor::get_num_devices() > 0) {
         exec = gko::HipExecutor::create(0, gko::OmpExecutor::create());
     } else {
-        std::cerr << "Usage: " << argv[0] << " [executor]" << std::endl;
+        std::cerr << "Usage: executable [executor]" << std::endl;
```

Comment:
What is the purpose of these changes? Do we test for the usage message somewhere?

Reply:
The usage is being printed when all the arguments are not being passed. And `argv[0]` prints the executable with the whole path.

Reply:
I know, but in a correct execution of these examples, the usage message should never be printed, right? I'm trying to understand why this change is necessary.

Reply:
For most examples, yes. Some examples, like the [`three-pt-stencil-solver`](https://github.com/ginkgo-project/ginkgo/blob/develop/examples/three-pt-stencil-solver/three-pt-stencil-solver.cpp#L235) and a few others, the usage gets printed when the coefficients are not specified. So, I just wanted to make everything uniform.

Reply:
Makes sense, so unless we match the output more fuzzily, these specific examples will have to be modified.

Reply:
Yes, exactly.

Reply:
But the rest of the changes could be reverted then?

Reply:
Yes, I can do that. I just wanted the output to the uniform across all the examples, but i can revert the changes for the rest of the examples.